<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckMediaFilesDialog</class>
 <widget class="QDialog" name="CheckMediaFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check Media Files</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="commandClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>The following list contains all the Media Recordings in the active Corpus. Click on the Check Media Files button to begin the procedure: this will ensure that the associated files are accesible, and will check whether they have been modified since they were added to the corpus. In case of modified files, Praaline will attempt to read their technical specifications.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCommandLinkButton" name="commandProcess">
     <property name="text">
      <string>Check Media Files</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>483</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QTreeView" name="treeviewRecordings"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeviewRecordings</tabstop>
  <tabstop>commandProcess</tabstop>
  <tabstop>commandClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
